Day3/Java/HammingDistance1.java

```diff
@@ -20,8 +20,13 @@ public static void main(String[] args) {
 					k+ = 1;                       // if same character is not found change in counter variable k by 1
 				
 			}
-		}
+		
 		System.out.println("Hamming Distance is " + k);   // printing the hamming distance
+		}
+		else
+		{
+		System.out.println("Hamming Distance is 1"); // if both strings are unequal then hamming distance is eaqual to 1
```

Comment:
Hamming Distance is undefined for strings of unequal length. You can check the official solution [here](https://github.com/CodeToExpress/dailycodebase/blob/master/Day3/JavaScript/hamming.js).
```suggestion
		System.err.println("Hamming Distance is undefined"); // if both strings are unequal then hamming distance is undefined
```